daffodil-core/src/main/scala/org/apache/daffodil/core/runtime1/TermRuntime1Mixin.scala

```diff
@@ -229,10 +230,16 @@ trait TermRuntime1Mixin { self: Term =>
       // term itself has not provided a match to the incoming event, and we need to
       // see if things following the term provide a match.
       //
-      case stb: SequenceTermBase =>
+      case stb: SequenceTermBase => {
         followingLexicalSiblingStreamingUnparserElements
-      case _ =>
-        possibleSelfPlusNextLexicalSiblingStreamingUnparserElements
+      }
+      case _ => {
```

Comment:
```case _ ``` leaves something to be desired here. 

I think the possibilities are ElementBase or ChoiceTermBase.

Can we do 

```case _: ElementBase | _: ChoiceTermBase => ... ```